### PR TITLE
Support soft cross entropy loss

### PR DESCRIPTION
**Pull request by Junjun2016**
Support soft cross entropy loss

### CODE REVIEW

**Comment by yiminglin-ai** [comment]
How is the performance compared with standard (hard) CrossEntropy?
